packages/pipeline/src/parsers/events/exchange_events.ts

```diff
@@ -59,16 +59,26 @@ export function _convertToExchangeFillEvent(eventLog: LogWithDecodedArgs<Exchang
     exchangeFillEvent.takerFeePaid = eventLog.args.takerFeePaid;
     exchangeFillEvent.orderHash = eventLog.args.orderHash;
     exchangeFillEvent.rawMakerAssetData = eventLog.args.makerAssetData;
-    exchangeFillEvent.makerAssetType = makerAssetType;
+    exchangeFillEvent.makerAssetType = convertAssetProxyIdToType(makerAssetData.assetProxyId as AssetProxyId);
     exchangeFillEvent.makerAssetProxyId = makerAssetData.assetProxyId;
-    exchangeFillEvent.makerTokenAddress = makerAssetData.tokenAddress;
+    // HACK(abandeali1): this event schema currently does not support multiple maker/taker assets, so we store a null byte array when decoding assetData from the MultiAssetProxy
```

Comment:
It looks like this comment is wrong and we are actually storing the maker/taker address of the first asset.

Also, it should be possible to change the schema to support this. We would need to write a new migration and update the entity definitions. I don't think we need to do it in this PR since it's already so big, but I'll add a task in Asana.

Reply:
Asana task is here: https://app.asana.com/0/682899751319128/962283066289046

Reply:
Whoops, forgot to update the comment.